web/index.html

```diff
@@ -461,71 +526,52 @@ <h3>Used in</h3>
                     let query = this.searchQuery.toLowerCase()
                     return this.data.list.filter(x => x[1].toLowerCase().includes(query)).map(x => x[0]).slice(0, this.loaded)
                 },
-                craftingTree: function () {
-                    if (this.currentElement == "" || !this.data.data[this.currentElement]) {
-                        return [];
-                    }
-                    if (this.data.costs[this.currentElement] == 1) {
-                        return [];
-                    }
-
-                    let craftingList = [];
-                    let usedElements = new Set();
-                    let result = [];
-                    craftingList.push(this.currentElement);
-
-                    while (craftingList.length > 0) {
-                        let thisElement = craftingList.shift();
-                        if (usedElements.has(thisElement)) {
-                            continue;
-                        }
-                        usedElements.add(thisElement);
-
-                        if (Object.keys(this.data.craftingCache).length == 0 || !this.data.craftingCache[thisElement]) {
-                            return []
-                        }
-                        let [left, right] = this.data.craftingCache[thisElement]
-
-                        result.unshift({ A: left, B: right, C: thisElement });
-
-                        if (this.data.costs[left] != 1) {
-                            craftingList.push(left);
-                        }
-
-                        if (this.data.costs[right] != 1) {
-                            craftingList.push(right);
-                        }
-                    }
-                    return result
-                },
                 elementCount: function () {
                     return Object.keys(this.data.index).length - 1;
                 },
                 isDefaultElement: function () {
                     return this.data.costs[this.currentElement] == 1;
-                }
+                },
+                isMobile: function () {
+                    return (/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent));
+                },
             },
             methods: {
+                getChunk: function (value) {
```

Comment:
The = does not seem to be fully working either. Even though some recipes that broke with the + are now working, many others are not. In particular I have noticed that "Eclipse" still isn't importing properly